django/contrib/admin/utils.py

```diff
@@ -388,6 +388,8 @@ def display_for_field(value, field):
         return formats.number_format(value, field.decimal_places)
     elif isinstance(field, models.FloatField):
         return formats.number_format(value)
+    elif isinstance(field, models.FileField) and not value:
```

Comment:
Yes, though it's a slightly different behavior (<p></p> vs <p>(None)</p>). It's two few lines of code though :)

Actually, I just checked and the old behavior is <p></p>, so I'll change it.